File: example/src/multiple-files/Class1.cpp
/*
Copyright (c) 2017 Fernando Bevilacqua <dovyski@gmail.com>
Licensed under the MIT license.
*/

#include "cvui.h"
#include "Class1.h"

Class1::Class1() {
	checked = false;
}

void Class1::renderInfo(cv::Mat & frame) {
	cvui::window(frame, 10, 50, 100, 120, "Info");
	cvui::checkbox(frame, 15, 80, "Checked", &checked);
}

File: example/src/multiple-files/Class1.py
#
# This is a dummy class to illustrate the use of cvui in a project
# that contains multiple files.
#
# Copyright (c) 2018 Fernando Bevilacqua <dovyski@gmail.com>
# Licensed under the MIT license.
#

import cvui

class Class1:
	def __init__(self):
		self.checked = [False]

	def renderInfo(self, frame):
		cvui.window(frame, 10, 50, 100, 120, 'Info')
		cvui.checkbox(frame, 15, 80, 'Checked', self.checked)
